libs/vectorise/include/vectorise/fixed_point/fixed_point.hpp

```diff
@@ -729,7 +753,10 @@ template <typename T>
 constexpr FixedPoint<I, F>::FixedPoint(T n, meta::IfIsFloat<T> *)
   : data_(static_cast<typename FixedPoint<I, F>::Type>(n * ONE_MASK))
 {
-  assert(CheckNoOverflow(n * ONE_MASK));
+  if (CheckNoOverflow(n * ONE_MASK))
+  {
+    fp_state |= STATE_OVERFLOW;
+  }
   //    assert(CheckNoRounding<T>());
```

Comment:
is this a leftover comment?

Reply:
removed comment for now, will investigate the need for rounding separately